### ISSUE TITLE
Don't seem to be able to allow clients to use host commands?

### ISSUE THREAD

**Issue by Mudkipster** [Toemmsen96/ChatCommands#22]
No matter the settings in config or whether or not the host types /cohost (player) in the chat, a person who is client isn't able to activate any commands. And it just returns the message in console "player isn't host" (paraphrasing).

**Comment by Mudkipster** [comment]
I would like to apologize for not reading the rest of the issues this seems to have already been brought up. However I do still have issues with it which is unfortunate

I have tried this with "Has to be Host" and "Send Host Commands" Set to true and false because the wording doesn't make it clear whether or not true means it's active or not. Nothing has worked.

**Comment by Mudkipster** [comment]
To specify, the exact message it sends in chat after having CoHost used on you is 
"Received chat input: (command)"
"Not Host, trying to send command:(command)"
"Chat Message:  sent by: (player)"

**Comment by Toemmsen96** [comment]
> To specify, the exact message it sends in chat after having CoHost used on you is "Received chat input: (command)" "Not Host, trying to send command:(command)" "Chat Message: sent by: (player)"

Does the Host have the mod and have you as cohost enabled?

Also:
Im sorry i have not been able to test this functionality properly before, because most of my friends don't really play the game anymore, but i would like to make it work with some feedback.

**Comment by Mudkipster** [comment]
> > To specify, the exact message it sends in chat after having CoHost used on you is "Received chat input: (command)" "Not Host, trying to send command:(command)" "Chat Message: sent by: (player)"
> 
> Does the Host have the mod and have you as cohost enabled?
> 
> Also: Im sorry i have not been able to test this functionality properly before, because most of my friends don't really play the game anymore, but i would like to make it work with some feedback.

Yes, I created the modpack and sent it to the host via code, so everything should be exactly the same.

**Comment by Toemmsen96** [comment]
> > > To specify, the exact message it sends in chat after having CoHost used on you is "Received chat input: (command)" "Not Host, trying to send command:(command)" "Chat Message: sent by: (player)"
> > 
> > 
> > Does the Host have the mod and have you as cohost enabled?
> > Also: Im sorry i have not been able to test this functionality properly before, because most of my friends don't really play the game anymore, but i would like to make it work with some feedback.
> 
> Yes, I created the modpack and sent it to the host via code, so everything should be exactly the same.

Okay Thanks

**Comment by Toemmsen96** [comment]
i released a new update which is a complete rework of the mod. let me know if this fixed this for you.
